src/app/_mermaid_common/forms/directives/dateinput.dir.js

```diff
@@ -152,6 +156,20 @@ angular
             return null;
           };
 
+          $timeout(function() {
```

Comment:
I assume this is based on the numberinput validation etc. right? Don't like `$timeout` but if that's the way the others are done then this is fine.